properties/src/main/java/com/fasterxml/jackson/dataformat/javaprop/util/JPropPathSplitter.java

```diff
@@ -139,12 +139,14 @@ public JPropNode splitAndAdd(JPropNode parent,
     public static class CharPathOnlySplitter extends JPropPathSplitter
     {
         protected final char _pathSeparatorChar;
+        protected final char _escapeChar;
```

Comment:
Bit torn here, but perhaps `_pathSeparatorEscapeChar` would be most descriptive (given it only applies to path separator char, specifically, and is not general-purpose escape char).

Reply:
Done.